graf3d/gl/inc/TGLSdfFontMaker.h

```diff
@@ -0,0 +1,13 @@
+#ifndef ROOT_TGLSdfFontMaker
+#define ROOT_TGLSdfFontMaker
+
+#include "Rtypes.h"
+
+class TGLSdfFontMaker {
+public:
+    static int MakeFont(const char* ttf_font, const char* output_prefix, bool verbose=false);
+
+    ClassDef(TGLSdfFontMaker, 0); 
```

Comment:
One do not need `ClassDef` for classes not inherited from `TObject`, 
but one need to add this class into `LinkDef.h`

Reply:
Done.